### PR TITLE
 Make neutronExtraImage more generic 

### PR DESCRIPTION
**Pull request by lpiwowar**
The neutronExtraImage parameter in the tempest CR is tailored
specifically to neutron-tempest-plugin. Because of that it does not
allow details like flavor, ID or name associated with the given image.

This patch removes the neutronExtraImage and replaces it with the more
generic version of it - extraImages.

Depends-On: https://github.com/openstack-k8s-operators/ci-framework/pull/1647

### CODE REVIEW

File: .zuul.yaml [hunk 1]
```diff
@@ -57,6 +57,17 @@
           identity.v3_endpoint_type public
           service_available.swift false
           service_available.cinder false
+      cifmw_test_operator_tempest_workflow:
+        - stepName: 'full'
+        - stepName: 'single-test'
+          tempestRun:
+            includeList: |
+              tempest.api.compute.admin.test_flavors.FlavorsAdminTestJSON.test_create_flavor_using_string_ram
+      cifmw_test_operator_tempest_extra_image:
```
Comment (tosky) [inline .zuul.yaml:66]: cifmw_test_operator_tempest_extra_image or cifmw_test_operator_tempest_extra_images?

File: api/bases/test.openstack.org_tempests.yaml [hunk 2]
```diff
@@ -137,15 +137,78 @@ spec:
                           type: string
                       type: object
                     type: array
+                  extraImage:
```
Comment (tosky) [inline api/bases/test.openstack.org_tempests.yaml:140]: extraImages?

File: api/bases/test.openstack.org_tempests.yaml [hunk 3]
```diff
@@ -373,6 +436,74 @@ spec:
                         of tempest. Please refer to Please refer to https://docs.openstack.org/tempest/latest/
                         for the further explanation of the CLI parameters.
                       properties:
+                        ExtraImageType:
```
Comment (tosky) [inline api/bases/test.openstack.org_tempests.yaml:439]: (if the changes above are implemented, this should be ExtraImagesType)

File: .zuul.yaml (hunk 1 above)
Comment (lpiwowar) [reply .zuul.yaml]: thank you!
`cifmw_test_operator_tempest_extra_images`

File: api/bases/test.openstack.org_tempests.yaml (hunk 3 above)
Comment (lpiwowar) [reply api/bases/test.openstack.org_tempests.yaml]: good point, thanks!:)

File: api/bases/test.openstack.org_tempests.yaml (hunk 2 above)
Comment (lpiwowar) [reply api/bases/test.openstack.org_tempests.yaml]: good point, thank you!

**Comment by lpiwowar** [comment]
recheck

**Comment by lpiwowar** [comment]
recheck

File: .zuul.yaml [hunk 4]
```diff
@@ -63,6 +63,15 @@
           tempestRun:
             includeList: |
               tempest.api.compute.admin.test_flavors.FlavorsAdminTestJSON.test_create_flavor_using_string_ram
+      cifmw_test_operator_tempest_extra_images:
```
Comment (tosky) [inline .zuul.yaml:66]: cifmw_test_operator_tempest_extra_images

Comment (tosky) [reply .zuul.yaml:66]: Uops, I didn't read it correctly, sorry

**Review by kopecmartin — APPROVED** [review]
lgtm, the depends on is merged

**Pull request merged**